activestorage/app/javascript/activestorage/helpers.js

```diff
@@ -25,9 +25,8 @@ export function findElement(root, selector) {
 export function dispatchEvent(element, type, eventInit = {}) {
   const { disabled } = element
   const { bubbles, cancelable, detail } = eventInit
-  const event = document.createEvent("Event")
+  const event = new Event(type, {bubbles: bubbles || true, cancelable: cancelable || true})
```

Comment:
Not related to the change at hand: would it be possible to pass anything other than `true` for `bubbles` and `cancelable`? Seems like falsey values would always get overriden

Reply:
Aah, that is true. I just din't want to change from existing values and keep the code same as before 😅